web/src/components/organisms/Asset/AssetList/index.tsx

```diff
@@ -0,0 +1,206 @@
+import { useParams } from "react-router-dom";
+
+import Button from "@reearth-cms/components/atoms/Button";
+import CustomTag from "@reearth-cms/components/atoms/CustomTag";
+import DownloadButton from "@reearth-cms/components/atoms/DownloadButton";
+import Icon from "@reearth-cms/components/atoms/Icon";
+import {
+  ListToolBarProps,
+  ProColumns,
+  OptionConfig,
+  TableRowSelection,
+  TablePaginationConfig,
+} from "@reearth-cms/components/atoms/ProTable";
+import { UploadProps } from "@reearth-cms/components/atoms/Upload";
+import { Asset } from "@reearth-cms/components/molecules/Asset/asset.type";
+import AssetListHeader from "@reearth-cms/components/molecules/Asset/AssetList/AssetListHeader";
+import AssetListTable from "@reearth-cms/components/molecules/Asset/AssetList/AssetListTable";
+import { uuidToURL } from "@reearth-cms/utils/convert";
+import { dateTimeFormat, bytesFormat } from "@reearth-cms/utils/format";
+import { dateSort, numberSort, stringSort } from "@reearth-cms/utils/sort";
+
+import useHooks from "./hooks";
+
+const AssetList: React.FC = () => {
+  const { workspaceId, projectId } = useParams();
+  const {
+    assetList,
+    createAssets,
+    assetsPerPage,
+    navigate,
+    handleSearchTerm,
+    selection,
+    setSelection,
+    fileList,
+    setFileList,
+    uploading,
+    setUploading,
+    uploadModalVisibility,
+    setUploadModalVisibility,
+  } = useHooks(projectId);
+
+  const displayUploadModal = () => {
+    setUploadModalVisibility(true);
+  };
+  const hideUploadModal = () => {
+    setUploadModalVisibility(false);
+  };
+
+  const handleUpload = () => {
+    setUploading(true);
+    createAssets(fileList)
+      .catch(error => {
+        // TODO: notification
+        console.log(error);
+      })
+      .finally(() => {
+        setUploading(false);
+        setFileList([]);
+        hideUploadModal();
+      });
+  };
+
+  const uploadProps: UploadProps = {
+    name: "file",
+    multiple: true,
+    directory: false,
+    showUploadList: true,
+    accept: "image/*,.zip,.json,.geojson,.topojson,.shapefile,.kml,.czml,.glb",
```

Comment:
Can you move the accepted string outside of the component and export it

Reply:
Done!

Reply:
Great!